lib/util.ml

```diff
@@ -22,6 +22,13 @@ module Result = struct
     let ( >>= ) r f = match r with Ok x -> f x | Error _ as e -> e
 
     let ( >>| ) r f = match r with Ok x -> Ok (f x) | Error _ as e -> e
+
+    let ( >>! ) r f =
+      match r with
+      | Ok x -> f x
+      | Error (`Msg e) ->
+          Printf.eprintf "Fatal error: %s" e;
```

Comment:
How about mentioning MDX here? Since we're moving towards a model where mdx won't be called directly, might be worth mentioning where the error message comes from so that users won't have to figure that out for themselves.

Reply:
Also probably worth adding a `\n` at the end!

Reply:
done